cl_manycore/libraries/bsg_manycore_eva.cpp

```diff
@@ -309,9 +436,9 @@ static int default_eva_to_npa_dram(const hb_mc_manycore_t *mc,
         const hb_mc_config_t *cfg = hb_mc_manycore_get_config(mc);
 	dim = hb_mc_config_get_dimension_network(cfg);
 
-	xdimlog = default_get_x_dimlog(cfg);
-	xmask   = default_get_dram_x_bitidx(cfg);
-	shift   = default_get_dram_x_shift(mc);
+	xdimlog    = default_get_x_dimlog(cfg);
```

Comment:
Did you add tabs here?

Reply:
woops! how did that get in there :D fixed it!